### PR TITLE
feat(trial): Show create team modal after new signup

### PR DESCRIPTION
**Pull request by tristandubbeld**
Shows the create team modal after a new signup and dismisses the what's new modal. Only existing users get to see the what's new modal and new users get prompted to create a team.

### Testing

This is quite hard to test on dev environments so we'll have to properly test this on stream. We can bypass certain steps to test if the modal shows up though:

1. Open the dashboard
2. Open the browser devtools console
3. Past and execute the following:
```
localStorage.setItem('NUOCT22', true);
```
4. Refresh the page
5. It should show the create team modal
6. Refresh the page again
7. The create team modal should not show up

### CODE REVIEW

**Comment by lbogdan** [comment]
Build for latest commit https://github.com/codesandbox/codesandbox-client/pull/7051/commits/50b6e063337c2d72b362a26d23df7ce0d997889c is at https://pr7051.build.csb.dev/s/new.

File: packages/app/src/app/components/dashboard/InputText.tsx [hunk 1]
```diff
@@ -13,6 +13,8 @@ const StyledInput = styled.input`
   border: none;
   border-radius: 2px;
 
+  ${props => (props.isInvalid ? 'outline: 1px solid #EB5E5E;' : '')}
```
Comment (tristandubbeld) [inline packages/app/src/app/components/dashboard/InputText.tsx:16]: image

File: packages/app/src/app/pages/SignIn/SignIn.tsx [hunk 2]
```diff
@@ -22,8 +22,20 @@ export const SignIn = ({ redirectTo, onSignIn }: SignInProps) => {
     }
   }, [getPendingUser, pendingUserId]);
 
-  if (duplicateAccountStatus) {
-    return <DuplicateAccount provider={duplicateAccountStatus.provider} />;
+  /**
+   * 🚧 Utility to debug Duplicate Account
+   */
+  const DA_DEBUG = window.localStorage.getItem('DA_DEBUG') === 'ENABLED';
```
Comment (tristandubbeld) [inline packages/app/src/app/pages/SignIn/SignIn.tsx:28]: We now have the ability to debug the duplicate account view without having to jump through hoops creating new accounts. Right now this is out of scope for the next release, but we want to update the screen after.

**Review by tristandubbeld — COMMENTED** [review]
Added screenshots and comments explaining certain (unrelated-ish) changes.

File: packages/app/src/app/pages/Dashboard/index.tsx [hunk 3]
```diff
@@ -20,13 +20,14 @@ import { Sidebar } from './Sidebar';
...
 import { WhatsNew } from './Components/WhatsNew/WhatsNew';
+import { NUOCT22 } from '../SignIn/Onboarding';
 
 const GlobalStyles = createGlobalStyle({
   body: { overflow: 'hidden' },
 });
 
 const COLUMN_MEDIA_THRESHOLD = 1600;
-const wnoct22 = 'wnoct22'; // = whats new oct 22
+const WNOCT22 = 'WNOCT22'; // = whats new oct 22
```
Comment (olarclara) [inline packages/app/src/app/pages/Dashboard/index.tsx:30]: Changing this won't retrigger the modal after this goes to prod?

File: packages/app/src/app/pages/SignIn/Onboarding.tsx [hunk 4]
```diff
@@ -114,14 +121,23 @@ export const Onboarding = () => {
             }}
             value={newUsername}
             onChange={e => setNewUsername(e.target.value)}
+            required
+            isInvalid={pendingUser.valid === false}
```
Comment (olarclara) [inline packages/app/src/app/pages/SignIn/Onboarding.tsx:125]: _nit_: (should've been caught by prettier)

```suggestion
            isInvalid={pendingUser.valid === false}
            required
```

**Review by olarclara — APPROVED** [review]

File: packages/app/src/app/pages/Dashboard/index.tsx (hunk 3 above)
Comment (tristandubbeld) [reply packages/app/src/app/pages/Dashboard/index.tsx:30]: Oh, you're right 🧠! Let me undo this change.

File: packages/app/src/app/pages/SignIn/Onboarding.tsx [hunk 5]
```diff
@@ -114,14 +121,23 @@ export const Onboarding = () => {
...
           />
 
+          {!pendingUser.valid ? (
+            <Text size={3} variant="danger">
+              Sorry, that username is already taken.
+            </Text>
```
[the remark this answers is not in the document]
Comment (tristandubbeld) [reply packages/app/src/app/pages/SignIn/Onboarding.tsx:131]: It should, I'll add it.

**Pull request merged**